img/t/trendmicro.com.svg

```diff
@@ -0,0 +1 @@
.687l1.417-.357M268.836 25.797c0 .179.004.355.012.529.01.174.025.307.05.4h2.586v-.738c0-.483-.11-.87-.33-1.158-.222-.289-.586-.434-1.095-.434-.383 0-.682.12-.899.357-.216.238-.324.586-.324 1.044zm-1.337 2.674a19.025 19.025 0 0 1-.096-.63c-.03-.225-.054-.45-.07-.675a16.701 16.701 0 0 1-.052-1.241c0-.416.039-.825.116-1.229.076-.403.208-.764.394-1.082.188-.319.444-.573.77-.764.328-.192.738-.287 1.23-.287.73 0 1.324.17 1.782.51.459.34.776.793.955 1.362l.395-.624 3.4-2.037v2.011l-3.488 2.026-.167.916h3.655v1.744h-8.824M271.788 18.896c.45 0 .864-.039 1.241-.115.379-.076.703-.193.975-.35a1.79 1.79 0 0 0 .637-.604 1.64 1.64 0 0 0 .228-.88c0-.619-.241-1.098-.725-1.438-.484-.34-1.27-.51-2.356-.51-.433 0-.836.036-1.21.108-.373.073-.7.185-.98.338-.28.153-.498.352-.655.599-.158.246-.236.547-.236.903 0 .62.248 1.1.745 1.44.496.34 1.276.509 2.336.509zm0 1.834c-1.485 0-2.63-.327-3.43-.98-.803-.655-1.205-1.588-1.205-2.803 0-.636.111-1.192.331-1.667.221-.475.535-.87.943-1.185a4.122 4.122 0 0 1 1.464-.7 7.32 7.32 0 0 1 1.897-.23c1.487 0 2.63.33 3.432.988.803.658 1.203 1.59 1.203 2.794 0 .646-.11 1.206-.331 1.682-.22.475-.535.867-.941 1.177-.408.31-.896.541-1.466.695a7.318 7.318 0 0 1-1.897.229M69.053 13.378h31.506v6.466H88.976v32.444h-8.341V19.844H69.053v-6.466M148.813 13.378h27.902v6.466h-19.551v9.148h17.99v6.462h-17.99V45.82h20.139v6.467h-28.49v-38.91M208.012 52.288l-14.287-25.874-.108.116v25.758h-7.806v-38.91h10.341l14.205 25.426.108-.108V13.378h7.807v38.91h-10.26M239.807 45.818h-4.323V20.152h4.323c9.904-.11 11.362 6.484 11.362 12.68 0 6.201-1.458 13.105-11.362 12.986zm.573-32.433h-13.241v38.899h13.241c12.133.11 19.673-5.927 19.673-19.452 0-13.524-7.54-19.562-19.673-19.447M283.35 13.674h-1.422v3.771h-.563v-3.771h-1.408v-.505h3.392v.505M288 17.445h-.539v-3.58l-1.242 3.58h-.508l-1.223-3.64-.013 3.64h-.547v-4.28h.848l1.008 3.029c.008.01.044.125.115.355.046.134.076.227.084.279l.244-.686 1.018-2.976H288v4.28M115.395 20.042h9.86c3.308.047 5.387 1.24 5.748 4.011 0 .017.003.033.006.047.01.096.016.19.022.285.008.096.01.193.016.286.003.056.006.117.006.172.003.069.005.138.003.207-.034 3.376-2.18 5.17-5.8 5.22h-9.86Zm17.063 15.84c4.331-1.59 7.426-5.786 7.426-10.714 0-.588-.044-1.162-.13-1.725-.54-3.6-2.467-6.355-5.705-8.247-.454-.241-2.826-1.754-10.057-1.754h-.009c-.14.003-.279-.006-.417 0h-16.513v38.906h8.345V36.903h8.822s5.455 11.986 6.157 15.445h10.268l-8.187-16.466"/><path fill="#d91920" d="M61.225 30.614c0 16.907-13.705 30.613-30.612 30.613C13.706 61.227 0 47.52 0 30.614S13.706 0 30.613 0s30.612 13.706 30.612 30.613"/><path fill="#fff" d="M17.128 30.467c-4.442 8.697-3.854 11.459-3.088 13.612 2.062 5.796 13.204 4.17 24.181-4.405C49.2 31.097 54.225 19.19 49.45 13.076c-4.156-5.318-14.505-4.374-24.27 1.575 12.19-7.8 25.011-9.245 29.992-2.872 5.6 7.164-.77 21.495-14.226 32.006C27.489 54.3 12.042 57.013 6.443 49.85c-3.521-4.506-2.668-11.814 2.065-19.257H4.93l.847-1.976s6.417-1.294 12.886-7.17c.014-.013 2.602-.018 2.602-.018l-3.224 6.965h5.291l-1.002 2.099-5.201-.025"/></svg>
```

Comment:
What you've added is more akin to a company logo rather than an icon. Here's one I think would fit better:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 90 90"><circle cx="45" cy="45" r="45" fill="#d71920"/><path fill="#fff" d="M65.26 11.36c-8.89.21-19.48 4.04-29.33 10.4 10.53-6.28 21.92-9.07 30-6.57 12.58 3.92 12.33 19.26-.57 34.25-7.84 9.12-18.56 16.01-28.58 19.13-4.3 1.22-8.3 1.45-11.5.46-5.2-1.62-6.13-5.94-5.18-11.73 1.23-5.19 4.86-12.44 4.86-12.44l7.73-.02 1.43-3.08-7.75.02 4.82-10.33h-3.73l-.2.01A42.32 42.32 0 0 1 8.3 42.03L6.99 44.9l5.4-.01c-1.29 2.54-2.76 5.15-3.7 7.42l-.5 1.27c-2.26 6.19-2.5 11.83-.73 16.22a45.6 45.6 0 0 0 4.74 5.98c1.35.96 2.93 1.74 4.76 2.3C31.76 82.7 56.4 71.74 72 53.6c10.38-12.07 14.15-24.25 11.16-32.43a45.16 45.16 0 0 0-4.83-6.4 16.87 16.87 0 0 0-4.85-2.36 26.1 26.1 0 0 0-8.22-1.04z"/></svg>
```